.github/workflows/docker-hub-latest.yml

```diff
@@ -11,7 +11,8 @@ env:
   DOCKER_NAMESPACE: matrixdotorg
   PLATFORMS: linux/amd64
   # Only push if this is develop, otherwise we just want to build
-  PUSH: ${{ github.ref == 'refs/heads/develop' }}
+  # On a PR github.ref is the target branch, so don't push for that either
+  PUSH: ${{ github.ref == 'refs/heads/develop' && github.event_name != 'pull_request' }}
```

Comment:
Hm, the jobs might need a `env.PUSH == 'true'` perhaps?

Reply:
ah, indeed, since PUSH is an env var.